src/main/kotlin/com/terraformation/backend/customer/db/ParentStore.kt

```diff
@@ -137,13 +139,21 @@ class ParentStore(private val dslContext: DSLContext) {
   fun getOrganizationId(speciesId: SpeciesId): OrganizationId? =
       fetchFieldById(speciesId, SPECIES.ID, SPECIES.ORGANIZATION_ID)
 
+  fun getOrganizationId(submissionId: SubmissionId): OrganizationId? {
+    val projectId = getProjectId(submissionId)
+    return projectId?.let { getOrganizationId(it) }
+  }
```

Comment:
Rather than two separate fetches, this could take advantage of jOOQ's implicit join feature. We do this in other places in this file such as `getFacilityConnectionState()`. It'd look something like
```kotlin
fetchFieldById(submissionId, SUBMISSIONS.ID, SUBMISSIONS.projects.ORGANIZATION_ID)
```